operate/common/src/main/java/io/camunda/operate/JacksonConfig.java

```diff
@@ -28,7 +28,7 @@
 import org.springframework.context.annotation.Configuration;
 import org.springframework.http.converter.json.Jackson2ObjectMapperBuilder;
 
-@Configuration
+@Configuration("operateJacksonConfig")
```

Comment:
Why is this necessary?

Reply:
See the commit message in 844faf70584dbd6132131d2bd9f9cef4616bbf37, it's necessary because we want to load both and not giving them a name will result in a conflict that prevents Spring startup.